aggsender/types/types.go

```diff
@@ -270,12 +270,22 @@ func (c *CertificateHeader) IsClosed() bool {
 	return c.Status.IsClosed()
 }
 
+// ElapsedTimeSinceCreationString returns the time elapsed since the certificate was created as a string
+func (c *CertificateHeader) ElapsedTimeSinceCreationString() string {
+	t := c.ElapsedTimeSinceCreation()
+	if t == 0 {
+		return NAStr
+	}
+
+	return t.String()
+}
+
 // ElapsedTimeSinceCreation returns the time elapsed since the certificate was created
-func (c *CertificateHeader) ElapsedTimeSinceCreation() string {
+func (c *CertificateHeader) ElapsedTimeSinceCreation() time.Duration {
 	if c == nil || c.CreatedAt == 0 {
-		return NAStr
+		return 0
 	}
-	return time.Now().UTC().Sub(time.Unix(int64(c.CreatedAt), 0)).String()
+	return time.Now().UTC().Sub(time.Unix(int64(c.CreatedAt), 0))
 }
```

Comment:
In fact, this is the whole suggestion of how the function could look like. WDYT?

```suggestion
// ElapsedTimeSinceCreation returns the time elapsed since the certificate was created.
// If the certificate is nil or has no creation time, it returns 0 and false.
// If the creation time is in the future, it returns 0 and true.
func (c *CertificateHeader) ElapsedTimeSinceCreation() (time.Duration, bool) {
	if c == nil || c.CreatedAt == 0 {
		return 0, false
	}

	createdAt := time.Unix(int64(c.CreatedAt), 0).UTC()
	elapsed := time.Since(createdAt)

	if elapsed < 0 {
		return 0, true
	}
	return elapsed, true
}
```

Reply:
Fix: https://github.com/agglayer/aggkit/pull/920/commits/3c570b4a54247c5afb3e1d64df4d37bae31e124d